src/Mono.Android/Mono.Android.targets

```diff
@@ -168,6 +168,34 @@
         Command="jar cf &quot;$(OutputPath)mono.android.jar&quot; -C &quot;$(IntermediateOutputPath)jcw\bin&quot; ."
     />
   </Target>
+  <Target Name="_GenerateMonoAndroidDex16"
+      AfterTargets="_GenerateJavaCallableWrappers"
+      Condition="'$(JavacSourceVersion)' == '1.6'"
+      Inputs="$(OutputPath)mono.android.jar"
+      Outputs="$(OutputPath)mono.android.dex">
+    <Exec
+        Command="&quot;$(AndroidSdkDirectory)&quot;\build-tools\*\dx --dex --output=&quot;$(OutputPath)mono.android.dex&quot; &quot;$(OutputPath)mono.android.jar&quot;"
```

Comment:
I'm not entirely sure we want to use `*` like this. Yes, `android-toolchain.projitems` *should* be ensuring that there's only one possible match there, but *if there is more than one match*, things will break. (manually mucking around with the install? Future changes to permit manually specifying an Android SDK/NDK location w/o "management"?)

I *think* I'd be happier if we added an `$(XABuildToolsVersion)` property to `Configuration.props` and used that instead of `*`. Unfortunately this would need to be kept in-sync with `android-toolchain.projitems`, but the PR builder would be able to ensure that.